clients/src/test/java/org/apache/kafka/common/requests/ConsumerGroupDescribeRequestTest.java

```diff
@@ -47,6 +47,7 @@ void testGetErrorResponse() {
             ConsumerGroupDescribeResponseData.DescribedGroup group = response.data().groups().get(i);
             assertEquals(groupIds.get(i), group.groupId());
             assertEquals(Errors.forException(e).code(), group.errorCode());
+            assertEquals(e.getMessage(), group.errorMessage());
```

Comment:
I wonder if we could also add a unit test for the other request. Is it possible?